lib/graphql/relay/relation_connection.rb

```diff
@@ -23,9 +23,21 @@ def cursor_from_node(item)
         end
       end
 
+      def record_exists?(record)
+        if(defined?(ActiveRecord::Relation) && record.is_a?(ActiveRecord::Relation))
+          record.exists?
+        else
+          !record.empty?
```

Comment:
How does this change affect the Sequel::Dataset implementation? Does this run a different query than it used to?

Reply:
@newtonry is a colleague and we're looking at this together.

`empty?` is probably not what we're going for. Sidenote: I think `Sequel::Dataset`'s method for `EXISTS` is `exists` without `?` :( and we're looking to see if there's a small behavioral difference WRT `ActiveRecord::Relation`.

Reply:
(IMO it's fine to leave the non-AR code as it was)